unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/NftBreadcrumbComponentView.cs

```diff
@@ -73,6 +74,8 @@ public override void RefreshControl()
                 pooledObjects[view] = poolObj;
                 categoriesByIndex[i++] = view;
             }
+
+            container.ForceUpdateLayout();
```

Comment:
It's a worrying sign, what kind of bug you resolve with it?

Reply:
I've changed the implementation to avoid updating the layout if the component refreshes many times in a row